addons/portal/static/src/js/portal.js

```diff
@@ -90,7 +93,17 @@ publicWidget.registry.PortalHomeCounters = publicWidget.Widget.extend({
                 Object.keys(data).map(k => this.$("[data-placeholder_count='" + k + "']").text(data[k]));
             });
         });
-        return Promise.all(proms);
+        return Promise.all(proms).then(() => {
+            this.$('.o_portal_doc_spinner').remove();
+            this.$('[data-placeholder_count]').toArray().forEach(counter => {
+                if (counter.innerText !== '0' || this.countersToKeep.includes(counter.dataset['placeholder_count'])) {
```

Comment:
Same here actually, would be better to base this on counter result from counter rpc than to infer it from the innerText.